src/mocks/server_request.ts

```diff
@@ -16,7 +16,22 @@ export const MockServerRequest = function (
       }
     }
     if (options.body) {
-      request.headers.set("Content-Length", options.body.length.toString());
+      // Used to see if the body can be stringified, or if its a formdata object
+      if (typeof options.body === "string") {
+        request.headers.set("Content-Length", options.body.length.toString());
```

Comment:
if it's a string do we need `toString()` ?

Reply:
I don't think `length` is supposed to be converted to string.

Reply:
And I think it's in octets. So a the string `hello` has length 5 but the content length should be 40.
https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Content-Length

Reply:
Doesn’t .length on an array return an int? And judging by that article, I think it should be a number and not a string

Reply:
Yup. It should be a number. If you use `.toString()` you are converting to a string.

Reply:
I was confused on what data type the content length was, will fix it 👍